packages/services/test/kernel/ikernel.spec.ts

```diff
@@ -97,7 +97,17 @@ describe('Kernel.IKernel', () => {
       defaultKernel.pendingInput.connect((sender, args) => {
         if (!called) {
           called = true;
-          defaultKernel.sendInputReply({ status: 'ok', value: 'foo' });
+          defaultKernel.sendInputReply(
+            { status: 'ok', value: 'foo' },
+            {
```

Comment:
sendInputReply doesn't take a second argument, so I don't think passing a second argument in is going to do anything.

Reply:
Yes, I added a second argument `parent_header` to `sendInputReply` [here](https://github.com/jupyterlab/jupyterlab/pull/12376/files#diff-1acd3b371270660fed12e4595862cfb74ab7e973dd853b9caac92dcb34b6ca99R821).

Reply:
Ah, sorry, maybe I was just looking at one commit (from an email notification) rather than the whole PR. I'll rescind my comments and make sure I'm looking at the whole PR.